nominal/nominal.py

```diff
@@ -74,15 +76,21 @@ def get_user() -> User:
 
 
 def upload_tdms(
-    file: Path | str, name: str | None = None, description: str | None = None, *, wait_until_complete: bool = True
+    file: Path | str, name: str | None = None, timestamp_column: str | None = None,
+        timestamp_type: ts._AnyTimestampType | None = None, description: str | None = None,
```

Comment:
description should be left as the third arg so that it preserves back compat when called as a positional arg